src/views/homepage/index.tsx

```diff
@@ -1,8 +1,41 @@
 import { ReactElement } from 'react'
+import { IconSupport24 } from '@hashicorp/flight-icons/svg-react/support-24'
+import { IconHelp24 } from '@hashicorp/flight-icons/svg-react/help-24'
+import { IconUser24 } from '@hashicorp/flight-icons/svg-react/user-24'
 import BaseNewLayout from 'layouts/base-new'
+import PreFooter from './components/pre-footer'
 
 function Homepage(): ReactElement {
-  return <>Homepage</>
+  return (
+    <>
+      <PreFooter
+        heading="Looking for help?"
+        description="Aenean interdum pulvinar nunc et maximus. Etiam imperdiet mattis sapien id commodow Aenean interdum pulvinar nunc nean interdum pulvinar."
+        ctas={[
+          {
+            icon: (
+              <IconSupport24 color="var(--token-color-highlight-foreground-primary)" />
+            ),
+            heading: 'Support',
+            description: 'Open a support ticket',
+            link: '/',
+          },
+          {
+            icon: <IconHelp24 color="var(--token-color-packer-brand)" />,
+            heading: 'Forum',
+            description: 'Find your answer on the forum',
+            link: '/',
+          },
+          {
+            icon: <IconUser24 color="var(--token-color-nomad-brand)" />,
+            heading: 'Community',
+            description: 'Join our community',
+            link: '/',
+          },
+        ]}
+      />
+    </>
```

Comment:
Nit, non-blocking: is the wrapping `Fragment` needed here, or could we omit it?

Reply:
It will be once more components are added to the page. Also reduces some merge conflicts that tend to happen towards the end of a project from my experience.

Reply:
Cool cool - makes sense, I like that approach! 💖